routes/classes.js

```diff
@@ -30,6 +29,34 @@ var classes = require('../lib/classes');
 router.get('/', restUtils.find(classes));
 
 /**
+ * @api {get} /classes/my Returns all the Classes where
+ * the user participates.
+ * @apiName GetClasses
+ * @apiGroup Classes
+ *
+ *  @apiHeader {String} x-gleaner-user.
+ *
+ *
+ * @apiSuccess(200) Success.
+ *
+ * @apiSuccessExample Success-Response:
+ *      HTTP/1.1 200 OK
+ *      [
+ *          {
+ *              "_id": "559a447831b76cec185bf501"
+ *          },
+ *          {
+ *              "_id": "559a447831b76cec185bf511"
+ *          }
```

Comment:
Fixed in: https://github.com/e-ucm/rage-analytics-backend/pull/105/commits/7ccbeede8b109585c472b844271365b2860036ee